packages/boxel-motion/addon/behaviors/tween.ts

```diff
@@ -0,0 +1,50 @@
+import linear from '@cardstack/boxel-motion/easings/linear';
+
+import Behavior, { EasingToFramesArgument, FPS, timeToFrame } from './base';
+
+export type Easing = (t: number) => number;
+
+export interface TweenBehaviorOptions {
+  easing?: Easing;
+}
+
+export default class TweenBehavior implements Behavior {
+  easing: Easing;
+
+  constructor(options?: TweenBehaviorOptions) {
+    this.easing = options?.easing ?? linear;
+  }
+
+  *getFrames(options: EasingToFramesArgument) {
+    let { from, to, duration, delay = 0 } = options;
+
+    if (from === to) {
+      return [];
```

Comment:
One more

Reply:
This one is correct and on purpose.

Reply:
Actually, maybe in theory we should put this check lower? since we might still want to generate frames for the delay? Interesting...